### DISCUSSION TITLE
关于计算器的错误类型

### DISCUSSION THREAD

**Discussion by bsnmldb** [liusy58/NJU_AdvancedProgramming_SP22#26] [Q&A] [answered]
是否只有ppt上 操作符误用（只有不能连续的操作符？） 圆括号不匹配 除数为0 的问题？
比如有数字错误（"." ".0" "2.22" "2.2.2"）的问题？

**Comment by binbin00** [comment, marked as answer]
都有可能，请尽可能考虑周全，测试的时候会着重测试常见的错误。
